documentation/book/security.adoc

```diff
@@ -1,40 +1,43 @@
 == Security
 
-The Apache Kafka project supports data encryption by means of SSL/TLS protocol.
-It's possible to encrypt data transferred between brokers and clients and between brokers (intra cluster communication).
-Leveraging the SSL/TLS support, it's also possible to have mutual authentication with the connected client providing its own certificate for being authenticated by the Kafka cluster.
-The Cluster Operator is in charge to setting up the SSL/TLS infrastructure by means of certificates for providing the security.
+The Apache Kafka project supports data encryption by means of the SSL/TLS protocol.
+This makes it possible to encrypt data transferred between brokers (interbroker communication) and between clients and brokers.
+Leveraging the SSL/TLS support, it is also possible to have mutual authentication, where the Kafka broker authenticates the client's certificate, and the client authenticates the server's certificate.
```

Comment:
"authenticates the client's certificate" sounds strange to me ...
It should be more "authenticates the client" or "verifies/validates the certificates" , or ?